README.md

```diff
@@ -2,16 +2,18 @@
 This module contains some tools to make a simple speaker verification.
 
 You can download it with PyPI:
-```
+```r
 $ pip install speaker-verification-toolkit
 ```
 
-To use in your own projects:
+To import and use in your own projects:
 ```python
 import speaker_verification_toolkit.tools as svt
+
+#   svt.some_function(...)
```

Comment:
Poderia colocar um exemplo básico. Acredito que quem for procurar a documentação já sabe como utilizar o python. A curiosidade está em como o pacote funciona.

Reply:
Aqui a ideia era mostrar como o import deve ser feito. Isso é só pra poupar o tempo de quem for usar, não tendo q bater cabeça vendo como é o sistema de pastas do package. Mas irei adicionar um código de exemplo bem básico também.